packages/@glimmer-workspace/build/lib/config.js

```diff
@@ -96,47 +108,157 @@ export function typescript(pkg, config) {
   });
 }
 
-/** @type {['is' | 'startsWith', string[], 'inline' | 'external'][]} */
-const EXTERNAL_OPTIONS = [
-  ['is', ['tslib', '@glimmer/local-debug-flags'], 'inline'],
-  ['is', ['@handlebars/parser', 'simple-html-tokenizer', 'babel-plugin-debug-macros'], 'external'],
-  ['startsWith', ['.', '/', '#', '@babel/runtime/'], 'inline'],
-  ['startsWith', ['@glimmer/', '@simple-dom/', '@babel/', 'node:'], 'external'],
-];
+/**
+ *
+ * @param {ExternalSpecifier | undefined} specified
+ * @returns {readonly string[]}
+ */
+function normalizeExternalsSpecifier(specified) {
+  if (specified === undefined) {
+    return [];
+  } else if (typeof specified === 'string') {
+    return [specified];
+  } else {
+    return specified;
+  }
+}
+
+/** @type {Record<string, ExternalConfig>} */
+const PRESETS = {
+  'workspace:recommended': {
+    inline: {
+      is: ['tslib'],
```

Comment:
what are we doing with tslib? O.o

afaict, it's more a legacy thing, and folks use `@babel/runtime` for not-yet-shipped ecma features